charts/apisix-ingress-controller/templates/deployment.yaml

```diff
@@ -39,6 +39,8 @@ spec:
         {{- toYaml . | nindent 8 }}
       {{- end }}
       serviceAccountName: {{ include "apisix-ingress-controller.serviceAccountName" . }}
+      securityContext:
```

Comment:
Should add docs in README.

Reply:
@tokers 
I added the docs on README as requested